fastai/core.py

```diff
@@ -53,6 +53,8 @@ def noop(*args, **kwargs): return
 def split_by_idxs(seq, idxs):
     last = 0
     for idx in idxs:
+        if idx >= len(seq) or idx < len(seq) * -1:
```

Comment:
For future reference, you can simply this to (IIRC):

```
not (-len(seq) <= idx < len(seq))
```

Reply:
you're right i like that better - the condition is more obvious